config/tech-docs.yml

```diff
@@ -3,15 +3,15 @@ host: docs.developer.service.hmrc.gov.uk
 
 # Header-related options
 show_govuk_logo: true
-service_name: CTC Traders API service guide
-service_link: https://developer.service.hmrc.gov.uk/guides/common-transit-convention-traders-service-guide
+service_name: HMRC Developer Hub 
+service_link: https://developer.service.hmrc.gov.uk/
 
 # Links to show on right-hand-side of header
 header_links:
-  CTC Traders API Service Guide: /guides/CTC-traders-api-end-to-end-service-guide
-  Documentation: /api-documentation/docs/using-the-hub
-  Applications: /developer/applications
-  Support: /developer/support
+  CTC Traders API Service Guide: /
+  Documentation: https://developer.service.hmrc.gov.uk/api-documentation/docs/using-the-hub
+  Applications: https://developer.service.hmrc.gov.uk/developer/login
+  Support: https://developer.service.hmrc.gov.uk/developer/support
```

Comment:
Can you explain why these are absolute references?

Reply:
Sorted